Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -314,23 +324,46 @@ void OpenALStream::SoundLoop()
     // many silence samples.  These do not need to be timestretched.
     if (rate > 0.10)
     {
-      soundTouch.setTempo(rate);
-      if (rate > 10)
+      if (use_timestretching)
+      {
+        soundTouch.setTempo(rate);
+      }
+      else
+      {
+        alSourcef(ui_source, AL_PITCH, static_cast<ALfloat>(rate));
```

Comment:
Sure, but as the master branch always uses SoundTouch the sound is similar with the one I already recorded. XAudio2 from the master branch also sounds the same as the one I posted.
OpenAL on master branch:
https://youtu.be/V8AO9KDvUiQ

I also tested on OpenAL by disabling the OpenAL call to frequency-shift the audio but it sounds just like XAudio2... Maybe the resampling algorithm from Dolphin is not 100% mature yet? I guess I should test on Linux to check.